docker/validator/setup.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh
 
+set -ue
```

Comment:
This change also makes the build-arg variables required instead of silently building without them set.